tests/cpgrid/lookupdataCpGrid_test.cpp

```diff
@@ -175,14 +192,14 @@ BOOST_AUTO_TEST_CASE(one_lgr_grid)
 {
     // Create a grid
     Dune::CpGrid grid;
-    const std::array<double, 3> cell_sizes = {1.0, 1.0, 1.0};
-    const std::array<int, 3> grid_dim = {4,3,3};
+    const std::array<double, 3>& cell_sizes = {1.0, 1.0, 1.0};
+    const std::array<int, 3>& grid_dim = {4,3,3};
```

Comment:
I am not insisting on changing this back, but what is the reason behind changing this?

I think the compiler might now create a temporary to hold the array and the life time of it is prolonged until the constant reference goes out of scope. The temporary object is used via the reference below.

Previously an object was created on the stack and used directly below

Reply:
Thanks for the explanation. I thought I was preventing copying. It doesn't take long to go back to the old version. I'll do it. Thanks again!

Reply:
Please note that this is a bit different from capturing a return value by const reference as e.g. in
```
const auto& featureInLGR = lookUpDataLGR(elem, fakeLgrFeatures[elem.level()-1]);
```

Even here I would be unsure whether this is worth it or not. That depends a bit on the implementation of the function (does it result in return value optimization (RVO) or not?) and on the type returned (is a copy expensive or not?)